Directory.Build.targets

```diff
@@ -9,11 +10,93 @@
     <PublishTrimmed Condition=" '$(TargetFramework)' == 'net6.0-ios' or '$(TargetFramework)' == 'net6.0-maccatalyst' ">false</PublishTrimmed>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(SingleProject)' == 'true'">
+    <!-- Required - Enable Launch Profiles for .NET 6 iOS/Android -->
+    <_KeepLaunchProfiles>true</_KeepLaunchProfiles>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(SingleProject)' == 'true'">
+    <!-- Required - Setup Single Project features -->
+    <ProjectCapability Include="MauiSingleProject" />
+
+    <!-- Optional - Enables a list of TFM's and device categories in the debug menu -->
+    <!-- This allows easily toggling of debug target TFM by selecting the platform -->
+    <!-- If removed, Top level debug targets show as a list of devices for the selected TFM -->
+    <ProjectCapability Include="XamarinStaticLaunchProfiles" />
+  </ItemGroup>
+
+  <!-- HACK: for windows -->
```

Comment:
From here down is a bunch of hacks to get WinUI to build with multi-targeting. It is still a separate project for now, but the bits are there to support a single project.